schema_utils.py

```diff
@@ -236,8 +236,10 @@ def has_length_at_most(obj, max_value):
 
     @staticmethod
     def is_nonempty(obj):
-        """Returns True iff the given object (a string) is nonempty."""
-        return bool(obj)
+        """Always returns True so that an input field can be made required
```

Comment:
This isn't the correct place to do this. It results in the method not saying what it does on the tin, and that's a programming antipattern.

Where exactly is the default value being tested? We need to modify that test somehow, I think.